ferrocene/doc/evaluation-report/src/rustc/tool-analysis.rst

```diff
@@ -23,7 +23,8 @@ Installation
      - Risk
      - Mitigation
      - Detectable
-   * - .. id:: RUSTC_ERR_INSTALL_01
+   * - .. _rustc_install_error:
```

Comment:
Why is this a link? We should in case link to the ID we created.

Reply:
was not aware I could f4fe11fc5c0